packages/frontend/src/pages/api/scorer/index.ts

```diff
@@ -0,0 +1,59 @@
+import { environment } from '@/config/environment'
+import log from '@/utils/log'
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { SubmitAddressForScoringRequestSchema, SubmitAddressForScoringResponse } from '@/types/api/scorer'
+import { ApiErrorResponse } from '@/types/api/error'
+import { urls } from '@/constants/urls'
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  switch (req.method) {
+    case 'POST':
+      // POST /api/scorer
+      // Submit a user's address, with their signature, to passport for scoring
+      return submitAddressForScoring(req, res)
+    default:
+      res.status(405).json({
+        error: 'Method not allowed',
+      } satisfies ApiErrorResponse)
+  }
+}
+
+async function submitAddressForScoring(req: NextApiRequest, res: NextApiResponse) {
+  const requestBodyResult = SubmitAddressForScoringRequestSchema.safeParse(req.body)
+  if (!requestBodyResult.success) {
+    res.status(400).json({
+      error: `Could not parse request: ${requestBodyResult.error}`,
+    } satisfies SubmitAddressForScoringResponse)
+    return
+  }
+  const requestBody = requestBodyResult.data
+
+  let gtcResult
+  try {
+    const gtcResponse = await fetch(new URL('/registry/v2/submit-passport', urls.gtcScorerApiBaseUri).href, {
+      method: 'POST',
+      headers: {
+        'X-API-KEY': environment.gtcScorerApiKey,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        scorer_id: environment.gtcScorerId,
+        address: requestBody.userAddress,
+        signature: requestBody.signature,
+        nonce: requestBody.nonce,
+      }),
+    })
+    gtcResult = await gtcResponse.json()
+    if (gtcResponse.status !== 200 || gtcResult.error) {
```

Comment:
Do we need to check for `gtcResult.error`? In the documentation, there is no such field in the response